packages/block-editor/src/components/image-size-control/index.js

```diff
@@ -67,7 +78,7 @@ export default function ImageSizeControl( {
 							min={ 1 }
 							onChange={ ( value ) =>
 								onChange( {
-									height: parseInt( value, 10 ),
+									height: normalizeDimension( value ),
```

Comment:
Maybe we can just use ternary here and check for "falsy" values. Example: `value ?  parseInt( value, 10 ) : undefined`